### ISSUE TITLE
Question on using bam by chr  for EH analysis

### ISSUE THREAD

**Issue by methionine23** [Illumina/ExpansionHunter#190]
Hello, I wonder if anyone has experiences and thoughts on running EH using a sub bam (eg  bams of one chr or an assembly for many samples). Certainly need to figure out MAP 0 and off target reads/region. Will that work?

**Comment by andreasssh** [comment]
Do you mean that you first align reads on all chromosomes and then subsetting a chromosome/gene/etc. from that file and running EH on the subsetted BAM?

**Comment by methionine23** [comment]
yes, also similar for a situation when I have bam per chr (or contig) instead of as in one bam. Any suggestions?

**Comment by andreasssh** [comment]
That's fine, works well. Only caveat is that if you want to use off-target regions then indeed, can't use reads aligned on a different chromosome if it is not present in your BAM. However, for expanded alleles you can still get a repeat estimate somewhere between read and fragment length (without using offtargets), which you can then investigate further and run on a full BAM if possible...
